components/omega/src/infra/logging.h

```diff
@@ -0,0 +1,108 @@
+// Omega logging header file.
+
+#ifndef OMEGA_LOG_H
+#define OMEGA_LOG_H
+
+#pragma once
+
+void omega_log_init();
```

Comment:
As in the implementation file, we want to wrap these in the OMEGA namespace and we can remove the omega_ prefix and change to camel case LogInit